src/core/TabPanel/TabPanel.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import { useTheme } from '@material-ui/core/styles';
+import { makeStyles } from '@material-ui/core/styles';
+
+interface TabPanelProps {
```

Comment:
The Base props needs to be extended here in order to leverage the power of the default Tab component

Reply:
@S-ayanide TabPanel is not available in the MUI core. Where can I extend its Base props?

Reply:
You won't need to create a TabPanel.tsx component separately, instead just use simple `Tabs` from MUI